WordPressKitTests/Utilities/HTTPRequestBuilderTests.swift

```diff
@@ -205,6 +205,15 @@ class HTTPRequestBuilderTests: XCTestCase {
         try XCTAssertEqual(builder.query(name: "foo", value: "bar").build().url?.query, "locale=zh&foo=bar")
     }
 
+    func testDefaultQueryExistsInOriginalURL() throws {
```

Comment:
What do you think of 

```suggestion
    func testDoesNotOverriedQueryItemInOriginalURL() throws {
```